appinfo/routes.php

```diff
@@ -123,5 +123,7 @@
 		['name' => 'ApiColumns#createTextColumn', 'url' => '/api/2/columns/text', 'verb' => 'POST'],
 		['name' => 'ApiColumns#createSelectionColumn', 'url' => '/api/2/columns/selection', 'verb' => 'POST'],
 		['name' => 'ApiColumns#createDatetimeColumn', 'url' => '/api/2/columns/datetime', 'verb' => 'POST'],
+
+		['name' => 'Contexts#index', 'url' => '/api/3/contexts', 'verb' => 'GET'],
```

Comment:
Why are you using 2 here instead of 3? Do we usually move to a new version when there's a major database change?

Reply:
I'd stick with 2 here as long as we don't break existing endpoints. I like the approach talk is doing for api versioning by adding feature flags to capabilities for new feature apis https://github.com/nextcloud/spreed/blob/main/lib/Capabilities.php#L107-L108 Maybe we could also adapt that over here?

Reply:
Right, that was one point I was also thinking forth and back. I went with 3, for it is an addition, and seemed semantically be more appealing. But 2 is fine, I'll switch it to 2.